conda_lock/pypi_solver.py

```diff
@@ -214,8 +214,11 @@ def get_requirements(
 
             if op.package.source_type == "url":
                 url, fragment = urldefrag(op.package.source_url)
-                hash_type, hash = fragment.split("=")
-                hash = HashModel(**{hash_type: hash})
+                hash_splits = fragment.split("=")
```

Comment:
Oh, it looks like we [don't have coverage](https://app.codecov.io/gh/conda/conda-lock/blob/main/conda_lock%2Fpypi_solver.py#L217) for this case. :cry: Let's fix that...